.travis.yml

```diff
@@ -39,6 +39,18 @@ matrix:
     - php: 7.2
       env:
         - DEPS=latest
+    - php: 7.3
+      env:
+        - DEPS=lowest
```

Comment:
lowest on php 7.3 can be removed to match against develop

Reply:
develop should do both lowest and latest... if it's not, we'll need to add it.

Reply:
I commented at https://github.com/weierophinney/laminas-servicemanager/pull/1#issuecomment-626799552 about the reason lowest at php 7.3 removed

Reply:
I was able to workaround it by adding some logic to use later versions of phpunit and ocramius/proxy-manager. Since these are _testing_requirements, and not _library_ requirements, we can use later versions and still be testing the functionality properly. As you'll see now, tests are green!